android/app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
     package="com.imaworldhealth.bhima_stock">
     <uses-permission android:name="android.permission.INTERNET"/>
     <application
-        android:label="BHIMA Stock"
+        android:label="Bhima Stock"
```

Comment:
I think you can keep it in uppercase as it was: BHIMA, since it is an abbreviation